### PR TITLE
Added documentation for cc_emails

### PR DESCRIPTION
**Pull request by lscoates**

### CODE REVIEW

File: doculab/docs/api-customers.textile [hunk 1]
```diff
@@ -201,6 +202,7 @@ Feature: Chargify XML API Customers read/fetch
         <first_name>Joe</first_name>
         <last_name>Blow</last_name>
         <email>joe@example.com</email>
+        <cc_emails>`your value`</cc_emails>
```
Comment (drewblas) [inline doculab/docs/api-customers.textile]: "``your value`"  - Can't we put an example of a few comma separated values?

Comment (lscoates) [inline doculab/docs/api-customers.textile]: I've updated the examples to show actual values where applicable.

**Comment by drewblas** [comment]
@lscoates Looking at the diff, I'm still seeing a very large number of `<cc_emails>`your value`</cc_emails>`  Are some being left that way on purpose?

**Comment by lscoates** [comment]
I've updated all of the examples where it feels applicable. There are some examples where it doesn't fit the pattern to add in a value (where all of the customer values are set to "your value". 
), eg:
https://github.com/chargify/docs.chargify.com/blob/update_for_cc_emails/doculab/docs/api-subscriptions.textile#xml-update-usage-example

Let me know if that's okay to leave those. The alternative would be to add in values for all of the keys, which I can go through a do if that makes sense!

**Comment by drewblas** [comment]
:shipit:

**Pull request merged**